ui/app/mainui/Popups.qml

```diff
@@ -362,6 +362,7 @@ QtObject {
                 onEditDone: {
                     if (nickname !== newNickname) {
                         rootStore.contactStore.changeContactNickname(publicKey, newNickname)
+                        Global.contactRenamed(publicKey)
```

Comment:
There are signals in contacts module `contactNicknameChanged` and `contactUpdated` which can also trigger a change of localNickname and other names. Probably it would better to handle changes from status-go and ui in one place

Reply:
The problem is that not all people are contacts (e.g. added contacts), this I haven't figured out how to do.